src/snowcli/snow_connector.py

```diff
@@ -56,3 +77,52 @@ def _find_command_path():
         # Example: SNOWCLI.WAREHOUSE.STATUS
         return ".".join(["SNOWCLI", *ctx.command_path.split(" ")[1:]]).upper()
     return "SNOWCLI"
+
+
+def load_pem_to_der(private_key_path: str) -> bytes:
+    """
+    Given a private key file path (in PEM format), decode key data into DER
+    format
+    """
+    from cryptography.hazmat.backends import default_backend
+    from cryptography.hazmat.primitives.serialization import Encoding
+    from cryptography.hazmat.primitives.serialization import NoEncryption
+    from cryptography.hazmat.primitives.serialization import PrivateFormat
+    from cryptography.hazmat.primitives.serialization import load_pem_private_key
+
+    with open(private_key_path, "rb") as f:
+        private_key_pem = f.read()
+
+    private_key_passphrase = os.getenv("PRIVATE_KEY_PASSPHRASE", None)
```

Comment:
I wonder if `PRIVATE_KEY_PASSPHRASE` isn't too generic name. Maybe we should add some prefix to it. @sfc-gh-turbaszek WDYT?

Reply:
This is the name we use in documentation for python connector. Let's stick to it.

Reply:
That's what I was thinking, thanks!